src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -270,7 +270,7 @@ private void errorHandler(String error, EmbedBuilder embedBuilder, MessageReceiv
                     // The error was not already listed and no fix was found. Add some info about the error
 
                     for (StackLine line : exception.getLines()) {
-                        if (line.getStackPackage() != null && line.getStackPackage().startsWith("org.geysermc") && !line.getStackPackage().contains("shaded")) {
+                        if (line.getStackPackage() != null && line.getline() != null && line.getStackPackage().startsWith("org.geysermc") && !line.getStackPackage().contains("shaded")) {
```

Comment:
Compilation has failed due to this, as shown here: https://github.com/GeyserMC/GeyserDiscordBot/actions/runs/3571433737/jobs/6306919452

The method is `getLine()`, instead of `getline()`